src/main.rs

```diff
@@ -77,22 +85,31 @@ impl Default for BananaSpawnTimer {
     }
 }
 
-pub fn banana_movement(mut banana_query: Query<(&mut Transform, &Banana)>, time: Res<Time>) {
-    for (mut transform, _banana) in banana_query.iter_mut() {
-        let direction = Vec3::new(0.0, -1.0, 0.0);
+pub fn banana_movement(
+    mut query: Query<(
+        &mut Transform,
+        Option<&Banana>,
+        Option<&BananaBunch>,
+        &BananaPoints,
+    )>,
+    time: Res<Time>,
```

Comment:
In the future, I would like to add more objects that fall from the sky like power-ups, etc. I can see that this function definition might become an issue if we keep adding more objects. Is there any way we could abstract the idea of a "falling object" into a single component so that we don't have to continue adding additional parameters over time?